cmd/selfupdate.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of K9s
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"runtime"
+
+	"github.com/creativeprojects/go-selfupdate"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+func selfUpdate() *cobra.Command {
+	command := cobra.Command{
+		Use:   "selfupdate",
+		Short: "Update k9s binary",
+		Long:  "Update k9s binary",
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
I think we should define additional flags here:
1. The default  aka upgrade to the latest rev
2. Update to a given version 
3. List the last x releases and dates
4. Rollback to previous (default) or given version

Perhaps leveraging symlinks here could be interesting?? ie keep the last x on disk and just swap the link if present?? 
Given my release track record that could be handy... ;)
In this case we could offer a `purge` option to axe previous revs?

Reply:
This is great improvements. I forgot about rollbacks at all! I will add these options.